packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -64,6 +66,8 @@ BuildRequires: %{?scl_prefix}rubygem(rabl)
 BuildRequires: %{?scl_prefix}rubygem(rest-client)
 BuildRequires: %{?scl_prefix}rubygem(runcible) >= 2.11.0
 BuildRequires: %{?scl_prefix}rubygem(runcible) < 3.0.0
+BuildRequires: %{?scl_prefix}rubygem(pulpcore_client)
```

Comment:
You can't add a BuildRequires on a package that you add in the same PR. Our build process doesn't support this.

Reply:
ahhh, you commented previously (or maybe it was @evgeni) that i could combine these types of prs together (that had inter dependencies),  i will split it out

Reply:
Runtime requirements are ok (`Requires`) since we can combine those and run repoclosure over the set. Koji doesn't know about other scratch builds so it can't build. Copr can, but we had some performance issues with that.